src/components/SecondaryPanes/Breakpoints.js

```diff
@@ -190,20 +187,33 @@ class Breakpoints extends Component<Props> {
 
     const groupedBreakpoints = groupBy(
       sortBy([...breakpoints.valueSeq()], bp => bp.location.line),
-      bp => getBreakpointFilename(bp.source)
+      bp => bp.source.url
     );
 
     return [
       ...Object.keys(groupedBreakpoints)
-        .sort()
-        .map(filename => {
+        .sort((urlA, urlB) => urlA.split("/").pop() > urlB.split("/").pop())
+        .map(url => {
+          const split = getRawSourceURL(url).split("/");
+          const file = split[split.length - 1].split("?")[0];
```

Comment:
It would be nice if this were a helper method next to `getFilename` perhaps `getFilenameFromRawUrl`:

- the helper function is easier to document the rationale
- easier to unit test 
- can be re-used in other places